dev/ci/ci-compcert.sh

```diff
@@ -7,5 +7,5 @@ git_download compcert
 
 ( cd "${CI_BUILD_DIR}/compcert" && \
       ./configure -ignore-coq-version x86_32-linux && \
-      make && \
+      COQCOPTS='-native-compiler no -w -undeclared-scope -w -omega-is-deprecated' make && \
```

Comment:
That's not just disabling native-compile ;)
I don't mind lessening the warning spam personally though

Reply:
Their makefile contains `COQCOPTS ?= -w -undeclared-scope -w -omega-is-deprecated`, I just kept it.